build/mp4-muxer.d.ts

```diff
@@ -14,7 +16,7 @@ declare interface VideoOptions {
 	/**
 	 * The clockwise rotation of the video in degrees.
 	 */
```

Comment:
Adjust this

Reply:
Mentioned the transformation matrix now in the documentation, but I am not sure how thoroughly to mention it.  
Is it enough or should something like "UVW Mapping" with a Wikipedia link be mentioned?